src/main/resources/data/create/recipes/compat/alloy_forgery/zinc_from_ores_forging.json

```diff
@@ -0,0 +1,32 @@
+{
+"fabric:load_conditions": [
+   {
+      "condition": "fabric:all_mods_loaded",
+      "values": [
+        "alloy_forgery"
+      ]
+    }
+  ],
+  "type": "alloy_forgery:forging",
+  "inputs": [
+    {
+      "tag": "c:ores/zinc"
+    }
+  ],
+  "output": {
+    "id": "create:zinc_ingot",
+    "count": 2
+  },
+  "overrides": {
+    "2": {
+      "id": "create:zinc_ingot",
+      "count": 3
+    },
+    "3+": {
+      "id": "create:zinc_ingot",
+      "count": 4
+    }
+  },
+  "min_forge_tier": 1,
+  "fuel_per_tick": 5
+}
```

Comment:
All JSONs are missing newlines. minor nitpick, but since changes are needed I might as well mention it.